### PR TITLE
Update documentation for gensim.similarities.docsim

### PR DESCRIPTION
**Pull request by souravsingh**
Initial start at fixing the docstrings of docsim.py

### CODE REVIEW

File: gensim/similarities/docsim.py [hunk 1]
```diff
@@ -74,8 +74,7 @@
 
 
 class Shard(utils.SaveLoad):
-    """
-    A proxy class that represents a single shard instance within a Similarity
+    """A proxy class that represents a single shard instance within a Similarity
```
Comment (menshikh-iv) [inline gensim/similarities/docsim.py]: 120 character limit (not 80)
Also, for definitions, first line of docstring **must** end with `.`

File: gensim/similarities/docsim.py [hunk 2]
```diff
@@ -84,6 +83,14 @@ class Shard(utils.SaveLoad):
...
 
     def __init__(self, fname, index):
+        """
+        Parameters
+        ----------
+        fname : str
+            Path to corpus.
+        index : int
+            Index of sytactic unit in corpus.
+        """
```
Comment (menshikh-iv) [inline gensim/similarities/docsim.py]: missed `\n` (empty line for the end of each docstring).

File: gensim/similarities/docsim.py [hunk 3]
```diff
@@ -92,22 +99,32 @@ def __init__(self, fname, index):
         self.index = self.get_index()
 
     def fullname(self):
+        """Return fullname of the corpus.
+        """
```
Comment (menshikh-iv) [inline gensim/similarities/docsim.py]: if docstring < 120 characters, please write it in one line: `"""Return fullname of the corpus."""`

Comment (menshikh-iv) [reply gensim/similarities/docsim.py]: also, try to avoid 'return' word, use 'get' for example (here and everywhere).

Comment (menshikh-iv) [reply gensim/similarities/docsim.py]: besides, missing `Returns` section (here and everywhere).

Comment (souravsingh) [reply gensim/similarities/docsim.py]: Okay, will do the fixes

**Comment by menshikh-iv** [comment]
Ping @souravsingh, how is going with PR?

**Comment by menshikh-iv** [comment]
Because nothing happens here, this task delegated to @CLearERR, I close this PR.

**Comment by souravsingh** [comment]
@menshikh-iv I was busy with college work, so the whole thing went off my head. I will pick some other file to update if it is alright with you. 

Please let me know which file needs a docstrig update.

**Pull request closed without merging**